pkg/kubelet/container/image_puller.go

```diff
@@ -56,24 +59,18 @@ func shouldPullImage(container *api.Container, imagePresent bool) bool {
 	return false
 }
 
-// reportImagePull reports 'image pulling', 'image pulled' or 'image pulling failed' events.
-func (puller *imagePuller) reportImagePull(ref *api.ObjectReference, event string, image string, pullError error) {
-	if ref == nil {
-		return
-	}
-
-	switch event {
-	case "pulling":
-		puller.recorder.Eventf(ref, "Pulling", "Pulling image %q", image)
-	case "pulled":
-		puller.recorder.Eventf(ref, "Pulled", "Successfully pulled image %q", image)
-	case "failed":
-		puller.recorder.Eventf(ref, "Failed", "Failed to pull image %q: %v", image, pullError)
+// records an event using ref, event msg.  log to glog using prefix, msg, logFn
+func (puller *imagePuller) logIt(ref *api.ObjectReference, event, prefix, msg string, logFn func(args ...interface{})) {
+	logFn(fmt.Sprint(prefix, " ", msg))
```

Comment:
I think we already log the events in the log. How about only log if we cannot record an event?

Another thing is that we should have different verbose level for the logging.